src/main/java/graphql/language/AbstractNode.java

```diff
@@ -47,4 +47,15 @@ public SourceLocation getSourceLocation() {
         return list.stream().map(Node::deepCopy).map(node -> (V) node).collect(Collectors.toList());
     }
 
+    //TODO: Implement really in each Node
+    @Override
+    public ChildrenContainer getNamedChildren() {
+        return null;
+    }
+
+    //TODO: Implement really in each Node
+    @Override
+    public T withNewChildren(ChildrenContainer newChildren) {
+        return null;
+    }
```

Comment:
again make it abstract eventually to enforce this precondition

Reply:
yes, it was just temp

Reply:
it is actually in Node